src/api/db/structure.sql

```diff
@@ -232,7 +227,6 @@ CREATE TABLE `bs_request_actions` (
   `makeoriginolder` tinyint(1) DEFAULT '0',
   PRIMARY KEY (`id`),
   KEY `bs_request_id` (`bs_request_id`),
-  KEY `index_bs_request_actions_on_target_project` (`target_project`),
```

Comment:
that is obsolete why exactly?

Removing index_bs_request_actions_on_target_project_and_source_project sounds more logic to me. No idea where that one can be useful.